SU-HSDO/suhumsci
# ARCH-219 added webform paragraph type

# READY FOR REVIEW

# Summary
- Added webform paragraph type

# Needed By (Date)
- 8/1 (OMG August?!)

# Urgency
- low

# Steps to Test

1. checkout this branch
1. ensure to `composer update --prefer-source`
1. `blt drupal:sync --site=archaeology --partial`
1. create a webform
1. added that webform to a basic page via the webform paragraph type
1. ensure the webform displays.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
